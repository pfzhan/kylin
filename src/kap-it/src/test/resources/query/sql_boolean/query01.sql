-- test is false, is true, is not false, is not true

SELECT trans_id,
CASE WHEN IS_EFFECTUAL IS FALSE THEN 0 ELSE 1 END,
CASE WHEN IS_EFFECTUAL IS NOT TRUE THEN 0 ELSE 1 END
FROM TEST_KYLIN_FACT
WHERE (PRICE > 0) IS TRUE AND (ITEM_COUNT > 0) IS NOT FALSE

