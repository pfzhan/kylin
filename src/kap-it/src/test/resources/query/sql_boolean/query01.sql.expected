-- test is false, is true, is not false, is not true

SELECT trans_id,
CASE WHEN NOT IS_EFFECTUAL THEN 0 ELSE 1 END as case1,
CASE WHEN NOT IS_EFFECTUAL THEN 0 ELSE 1 END as case2
FROM TEST_KYLIN_FACT
WHERE PRICE > 0 AND ITEM_COUNT > 0